README.md

```diff
@@ -121,9 +125,12 @@ Configurations are the interface that developers interact with when using the Za
 Because configurations are the core of a Zanzibar application, we create a root directory to host configuration files when starting a Zanzibar application. There are a few typical directories and files under the root directory. Take [example-gateway](https://github.com/uber/zanzibar/tree/master/examples/example-gateway) for example:
 ```
 example-gateway                 # root directory
+├── adapters                    # config directory for modules of adapter module class
+│   └── rate-limiter            # config directory for an adapter named 'rate-limiter' 
```

Comment:
use a different example, on line 75 you are using rate-limiter as a middleware.

Reply:
Will change to log-publisher